### PR TITLE
fix: UserGuide.adoc - more commands + v2.0 markers

### LINKED ISSUE #42: Update UG

### PR DESCRIPTION
**Pull request by shiyao821**
closes #42

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -202,40 +219,61 @@ Format: `edit [u/NAME] [p/PHONE_NUMBER] [a/ADDRESS]`
...
+Format: `loan sn/BOOK_SN...`
 
 Examples:
 
-* `loan sn/B0041` +
+* `loan sn/B00041` +
 Loans the book with serial number B0041
-* `loan sn/B0201 sn/B2929 sn/B0203` +
+* `loan sn/B00201 sn/B02929 sn/B00203` +
 Loans the books with serial numbers B0201, B2929 and B0203
```
Comment (hoholyin) [inline docs/UserGuide.adoc]: We need to update this too

**Review by hoholyin — COMMENTED** [review]
Other than the minor description that is not changed, the rest looks good!

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -135,17 +137,17 @@ Examples:
...
-Deletes the book with serial number `sn/B0422`.
+* `delete sn/B00422` +
+Deletes the book with serial number `sn/B00422`.
 
 ==== Registering a new borrower: `register`
 
-Registers a new borrower to the library records. A unique ID associated with the borrower will automatically be generated. +
-Format: `register u/NAME p/PHONE_NUMBER a/ADDRESS`
+Registers a new borrower to the library records. A unique ID associated with the borrower will automatically be generated and displayed. Borrowers are expected to know his ID in order for loans to be processed+
+Format: `register u/NAME p/PHONE_NUMBER e/EMAIL`
```
Comment (linyutinglyt) [inline docs/UserGuide.adoc:146]: Looks good!

**Review by linyutinglyt — COMMENTED** [review]
good job!

**Comment by seeziyang** [comment]
This can be merged! 👍

**Pull request merged**